tests/Unit/DataStructures/DataBox/Test_DataBoxDocumentation.cpp

```diff
@@ -0,0 +1,512 @@
+}
+
+/// [tagged_tuple_tags]
+struct Velocity2 {
+  using type = double;
+};
+struct Radius2 {
+  using type = double;
+};
+struct Density2 {
+  using type = double;
+};
+struct Volume2 {
+  using type = double;
+};
+/// [tagged_tuple_tags]
+
+double tagged_tuple_databox_1(){
+/// [tagged_tuple_databox_1]
+tuples::TaggedTuple<Velocity2, Radius2, Density2, Volume2> junior_databox{
+  4.0, 2.0, 0.5, 10.0};
+/// [tagged_tuple_databox_1]
+  return tuples::get<Density2>(junior_databox) *
+         tuples::get<Volume2>(junior_databox) *
+         tuples::get<Velocity2>(junior_databox) *
+         tuples::get<Velocity2>(junior_databox) /
+         tuples::get<Radius2>(junior_databox);
+}
+
+/// [tagged_tuple_mass_compute]
+template<typename... Tags>
+double mass_compute(const tuples::TaggedTuple<Tags...>& box) noexcept {
+
+  return tuples::get<Density2>(box) * tuples::get<Volume2>(box);
+}
+/// [tagged_tuple_mass_compute]
+
+/// [tagged_tuple_acceleration_compute]
+template<typename... Tags>
+double acceleration_compute(const tuples::TaggedTuple<Tags...>& box) noexcept {
+
+  return tuples::get<Velocity2>(box) * tuples::get<Velocity2>(box) /
+         tuples::get<Radius2>(box);
+}
+/// [tagged_tuple_acceleration_compute]
+
+/// [tagged_tuple_force_compute]
+template<typename... Tags>
+double force_compute(const tuples::TaggedTuple<Tags...>& box) noexcept {
+
+  const double mass = mass_compute(box);
+  const double acceleration = acceleration_compute(box);
+  return mass * acceleration;
+}
+/// [tagged_tuple_force_compute]
+
+/// [proper_databox_tags]
+struct Velocity3 : db::SimpleTag {
+  using type = double;
+  static std::string name() noexcept { return "Velocity3"; }
+};
+struct Radius3 : db::SimpleTag {
+  using type = double;
+  static std::string name() noexcept { return "Radius3"; }
+};
+struct Density3 : db::SimpleTag {
+  using type = double;
+  static std::string name() noexcept { return "Density3"; }
+};
+struct Volume3 : db::SimpleTag {
+  using type = double;
+  static std::string name() noexcept { return "Volume3"; }
+};
+struct Mass : db::SimpleTag {
+  using type = double;
+  static std::string name() noexcept { return "Mass"; }
+};
+/// [proper_databox_tags]
+
+double refined_databox_1(){
+/// [refined_databox]
+const auto refined_databox = db::create<
+    db::AddSimpleTags<
+      Velocity3, Radius3, Density3, Volume3>>(4.0, 2.0, 0.5, 10.0);
+/// [refined_databox]
+/// [refined_databox_get]
+const double velocity = db::get<Velocity3>(refined_databox);
+/// [refined_databox_get]
+const double radius = db::get<Radius3>(refined_databox);
+const double density = db::get<Density3>(refined_databox);
+const double volume = db::get<Volume3>(refined_databox);
+return density * volume *  velocity * velocity / radius;
+}
+
+/// [compute_tags]
+struct MassCompute : db::ComputeTag, Mass {
+  static std::string name() noexcept { return "MassCompute"; }
+  static constexpr auto function(const double& density, const double& volume){
```

Comment:
Define a free function called `mass_from_density` and then do `static constexpr auto function = &mass_from_density;`

apply liberally throughout